app/assets/javascripts/discourse/app/components/groups-form-interaction-fields.js

```diff
@@ -98,6 +98,9 @@ export default Component.extend({
     return (!isCreated && canCreate) || (isCreated && canAdmin);
   },
 
+  @discourseComputed("model.can_edit_group")
+  canEditGroup: reads("canEdit"),
```

Comment:
I am confused. Why both `discourseComputed` and `reads`?

Reply:
I think this is my mistake when going back and forth on this. Will fix. 🙏